app/controllers/activity_insight_oa_workflow/metadata_curation_controller.rb

```diff
@@ -14,6 +14,10 @@ def show
 
   def create_scholarsphere_deposit
     publication = Publication.find(params[:publication_id])
+    unless publication.can_deposit_to_scholarsphere?
+      flash[:warning] = I18n.t('activity_insight_oa_workflow.metadata_curation_list.cannot_be_deposited')
```

Comment:
When you were testing this, how did it go timing wise with the success notice? If someone double clicks, will the success pop up over the warning or will it be left on the warning?

Reply:
The warning shows up.

Reply:
Assuming they double click fairly quickly, the first click creates a successful deposit, and the second click triggers the warning, is it going to make users think that their deposit was entirely unsuccessful? Or am I missing something?

Reply:
Yeah, I'm conflicted on the wording too.  Technically, this could happen in more instances than just a double-click.  For instance if someone opened another tab for the submission form, then edited some of the metadata for the publication (rendering it no longer depositable), then we'd want this validation to stop that too.  This flash message is just very general for a user mistake that's hard to code for.

Reply:
So yes, the user will initially think it didn't submit, but if they check the scholarsphere deposits tab in the admin dashboard they'll see it deposited.  Also, they'll see it in scholarsphere.

Reply:
Yeah, that is tricky. If the double click isn't happening that often, we're probably good leaving it general, but if it's more frequent, we might want to consider adding something more in the warning text or even something preemptive by the button saying click once then wait

Reply:
Agreed